Doc/library/re.rst

```diff
@@ -786,13 +786,17 @@ form.
 
 .. function:: escape(string)
 
-   Escape all the characters in pattern except ASCII letters, numbers and ``'_'``.
+   Escape special characters in a string.
```

Comment:
Not sure what the style on naming for functions that take either str or bytes is, but can see argument for "...characters in string or bytes pattern".